### PR TITLE
challenge complete

### PR DESCRIPTION
**Pull request by paronicholas**
- [x] Top-level README “Table of Contents” is updated
- [x] Feature tasks for this challenge are completed
- [x] Unit tests written and passing
    - [x] “Happy Path” - Expected outcome
    - [x] Expected failure
    - [x] Edge Case (if applicable/obvious)
- [x] README for this challenge is complete
    - [x] Summary, Description, Approach & Efficiency, Solution, Resources
    - [x] Link to code
    - [x] Link to tests

### CODE REVIEW

File: code401challenges/src/main/java/utilities/Cat.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  public String getBreed () {
+    return breed;
+  }
+  
+  @Override
+  public int getAge () {
+    return age;
+  }
+  
+  public void setMedicalConcerns(String medicalConcern) {
```
Comment (evanslaton) [inline code401challenges/src/main/java/utilities/Cat.java]: This would normally be called `addMedicalConcerns` since you are adding a new concern to an arrayList. When you use `set` in the name of a method it usually means you are overwriting what was previously there with something new. So `setMedicalConcerns` would take in an array list and replace the previous array list with the new one.
